migrations/20161127144619-create-checker.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+module.exports = {
+  up: function(queryInterface, Sequelize) {
+    return queryInterface.createTable('Checkers', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      userid: {
```

Comment:
By convention sequelize uses ModelNameId convention for relationships, thus these should be UserId and TaskId. They should also have foreign key constraints check my https://github.com/izelnakri/nycda-blog-application/blob/refactoring/migrations/20161116101047-create-comment.js